packages/core/bootstrap/src/lib/rate-limit/index.ts

```diff
@@ -28,10 +28,10 @@ export const computeThroughput = (
   id: string,
 ): number => {
   // All observed in interval
-  const observedRequests = selectObserved(state, interval).filter((h) => !h.isWarmup)
+  const observedRequests = selectObserved(state, interval).filter((h) => !h.isCacheHit)
```

Comment:
I don't think this is the issue. Now you are just counting the requests that are going to the provider, but you want to take all of them in account.
I don't think you want to count warmup requests either